softwareDesign.md

```diff
@@ -0,0 +1,13 @@
+This is controversial. I think OO is itself an anti pattern. I recognise that's it's big in the software industry so i am doing my 
```

Comment:
You've definitely done your research here. OO is strange but it is one of the most popular programming paradigms.